src/client/components/atoms/itemList/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { Portfolio } from '../../../utils/Type';
+import getAssetUrl from '../../../utils/getAssetUrl';
+import Button from '../button';
+import './style.scss';
+
+type ItemListProps = {
+  item: Portfolio;
+};
+
+const ItemList = ({ item }: ItemListProps) => {
+  console.log('si', item);

```

Comment:
`no debugging code in PR`

, if prettier/eslint/ts complains about format, change it to a implicit return.